src/IOUService/web/ymmd.js

```diff
@@ -2,6 +2,8 @@
     set_up_send_button();
     load_current_user();
     load_sent_notes();
+    load_received_notes();
```

Comment:
Order of methods: Logically we do several things:
1. set up buttons
2. get current user
3. load notes.

The order of the calls should reflect this logic. Then we could even extract some more methods:
set_up_buttons(), load_notes()